src/NuGetForUnity/Editor/Models/NugetPackageV2.cs

```diff
@@ -54,5 +54,18 @@ public static NugetPackageV2 FromNupkgFile(string nupkgFilePath, NugetPackageSou
             package.DownloadUrl = nupkgFilePath;
             return package;
         }
+
+        /// <summary>
+        ///     Loads a <see cref="NugetPackageV2" /> from the .nuspec file at the given file-path.
+        /// </summary>
+        /// <param name="nuspecFilePath">The file-path to the .nuspec file to load.</param>
+        /// <param name="packageSource">The source this package was downloaded with / provided by.</param>
+        /// <returns>The <see cref="NugetPackageV2" /> loaded from the .nuspec file.</returns>
+        public static NugetPackageV2 FromNuspecFile(string nuspecFilePath, NugetPackageSourceV2 packageSource)
```

Comment:
I think this function is never used.

Reply:
You're right, it's not, but the function above (`FromNupkgFile`) was not used as well and I wanted to make it similar to how it is in `NugetPackageLocal` :) I will delete this one as per comment, and I can remove the one above as well.

Reply:
Yes remove both, the don't make sense as they need a remote package source, for a local file.